templates/checkout/terms.php

```diff
@@ -10,11 +10,18 @@
 	exit;
 }
 
-if ( wc_get_page_id( 'terms' ) > 0 && apply_filters( 'woocommerce_checkout_show_terms', true ) ) : ?>
-	<?php do_action( 'woocommerce_checkout_before_terms_and_conditions' ); ?>
+$terms_page_id = wc_get_page_id( 'terms' );
+$terms         = get_post( $terms_page_id );
+$terms_content = apply_filters( 'the_content', $terms->post_content );
```

Comment:
Better change `apply_filters( 'the_content', $terms->post_content )` to `wc_format_content( $terms->post_content )`, since `the_content` can be used for several plugins and can start load sharing buttons, related pages and other things inside the terms box.